sccp/logisticReg.py

```diff
@@ -5,3 +5,64 @@
 from sklearn.model_selection import StratifiedGroupKFold
 from sklearn.metrics import roc_auc_score
 
+def testPf2Ranks(pfx2_data, condition_labels_all, ranks_to_test,
```

Comment:
@pascoesean What does the input cv_group alter testPf2ranks? For these functions in logisticReg.py could you also add descriptions for each input and "why" you chose each automatic input as well? Like why penalty l1, what is penalty type, etc.